pkg/kubelet/dockertools/kube_docker_client.go

```diff
@@ -380,7 +381,14 @@ func (d *kubeDockerClient) RemoveImage(image string, opts dockertypes.ImageRemov
 	if ctxErr := contextError(ctx); ctxErr != nil {
 		return nil, ctxErr
 	}
-	return resp, err
+	if err != nil {
```

Comment:
dockertools is about to remove in next release, so we don't need to change its codes.